lib/broadside/target.rb

```diff
@@ -52,6 +53,7 @@ def initialize(name, options = {})
         dir = Broadside.config.config_file ? Pathname.new(Broadside.config.config_file).dirname : Dir.pwd
         env_file.expand_path(dir)
       end
+      @family = "#{Broadside.config.application}_#{@name}"
```

Comment:
the only thing that makes me hesitate about using a `@family` instead of a `def family` is that theoretically the gem could be reconfigured mid flight and `Broadside.config.application` could change, and that shouldn't break stuff.

 i'm in favor of `def family` with the the same output